app/models/application_record.rb

```diff
@@ -75,6 +75,23 @@ def pending?
     !reviewed
   end
 
+  def save_for_later(date = nil, note = nil, mail = false)
+    update_attributes(saved_for_later: true,
+                      date_for_later: date,
+                      note_for_later: note)
+    JobappsMailer.send_note_for_later(self).deliver_now if mail
+  end
+
+  def unsave
+    update_attributes(saved_for_later: false,
```

Comment:
Perhaps we should also clear out `note_for_later`.  What happens if we unsave and then resave?